src/library_webgpu.js

```diff
@@ -397,9 +397,9 @@ var LibraryWebGPU = {
 
     // Map from enum string back to enum number, for callbacks.
     Int_BufferMapState: {
-      'unmapped': 0,
-      'pending': 1,
-      'mapped': 2,
+      'unmapped': 1,
+      'pending': 2,
+      'mapped': 3,
```

Comment:
It looks like `Int_CompilationMessageType` might also be off by one:

```
system/include/webgpu/webgpu.h:    WGPUCompilationMessageType_Error = 0x00000001,
system/include/webgpu/webgpu.h:    WGPUCompilationMessageType_Warning = 0x00000002,
system/include/webgpu/webgpu.h:    WGPUCompilationMessageType_Info = 0x00000003,
```

How did this stuff ever work?  Or is it just not tested?

Reply:
A lot of these are pretty niche and most applications don't use them. It probably broke at some point when we updated the header (which shifted the numbers) but no one was using it.